.golangci.yml

```diff
@@ -268,7 +267,6 @@ linters-settings:
       - G503 # Import blocklist: crypto/rc4
       - G504 # Import blocklist: net/http/cgi
       - G505 # Import blocklist: crypto/sha1
-      - G601 # Implicit memory aliasing of items from a range statement
```

Comment:
was this meant to be removed?

Reply:
yeah, this is no longer relevant in go 1.22